registrations/serializers.py

```diff
@@ -404,6 +407,16 @@ def get_data_source(self, obj):
     def get_publisher(self, obj):
         return obj.publisher.id
 
+    def get_signup_url(self, obj):
+        def _get_signup_url_for_lang(lang):
+            return f"{settings.LINKED_REGISTRATIONS_UI_URL}/{lang}/registration/{obj.id}/signup-group/create"
+
+        return {
+            "en": _get_signup_url_for_lang("en"),
+            "fi": _get_signup_url_for_lang("fi"),
+            "sv": _get_signup_url_for_lang("sv"),
+        }
```

Comment:
```suggestion
        return {
            lang: f"{settings.LINKED_REGISTRATIONS_UI_URL}/{lang}/registration/{obj.id}/signup-group/create"
            for lang in ["en", "fi", "sv"]
        }
```

Reply:
Changed as suggested and added a test for the content